src/components/CodePreview/index.tsx

```diff
@@ -1,6 +1,7 @@
 import Highlight, { defaultProps } from "prism-react-renderer";
-import React, { CSSProperties, ReactNode } from "react";
-
+import React, { CSSProperties, ReactNode, useState, useEffect } from "react";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import startCase from "lodash/startCase";
```

Comment:
```suggestion
import startCase from "lodash.startcase";
```
Could this not be imported this way so you don't need to disable the eslint rule? Would need the types installed too

Reply:
This errors when even with the install of `@types/lodash`
<img width="664" alt="Screenshot 2024-02-15 at 15 05 15" src="https://github.com/mi6/ic-design-system/assets/113986285/2333acc0-a38b-447f-ad87-a553fc766d52">

If I keep it as `import startCase from "lodash/startCase";`, I get this error after running`npm run lint:fix`
<img width="811" alt="Screenshot 2024-02-15 at 15 15 21" src="https://github.com/mi6/ic-design-system/assets/113986285/7ceb6765-8661-4d30-9c9b-523a50df9096">
I decided against installing the entire `lodash` package to prevent increasing the size of the repo unnecessarily given I only need two methods from the package.

Reply:
I just tried this, switched over the import to my suggestion, but instead ran `npm install --save-dev @types/lodash.startcase --legacy-peer-deps`. Even with running `npm run lint:fix`, that should work?